ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -299,7 +299,6 @@ presubmits:
     labels:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-origin-master-e2e-aws-fips
-    optional: true
```

Comment:
If you're going to do this we don't really need to run e2e-gcp anymore.  I'd go ahead and remove e2e-gcp and just use e2e-aws-fips

Reply:
ok, we also have e2e-aws do we need both e2e-aws and e2e-aws-fips for origin?

Reply:
if you have e2e-aws just drop that one (in favor of this one) and leave e2e-gcp alone.  as long as e2e-aws is run all the time.  If e2e-aws is optional, we should make e2e-gcp optional instead, remove e2e-aws, and make e2e-aws-fips required

Reply:
ok looking closer we're testing the origin-e2e-aws-loki workflow with e2e-aws, so i did this:
- remove origin e2e-gcp 
- keep origin e2e-aws but make it optional
- make e2e-aws-fips blocking for origin

Reply:
@ingvagabund ok to remove the e2e-aws (testing w/ loki) here? if so, i'll remove it, if not no worries.

Reply:
after discussing on slack:
- e2e-aws-fips origin is blocking
- e2e-gcp left as/is
- e2e-aws removed (@ingvagabund will eventually add the loki logs to e2e-gcp)